squid_cloudwatch_filters/main.tf

```diff
@@ -26,8 +26,14 @@ locals {
     log_group_name = "${var.log_group_name_override == "" ? "${var.env_name}_/var/log/squid/access.log" : var.log_group_name_override}"
 }
 
+resource "null_resource" "log_group_found" {
```

Comment:
Any reason why we don't just go ahead and create the log group here? e.g. 

```
resource "aws_cloudwatch_log_group" "${local.log_group_name}" {
    name = "${local.log_group_name}"
}
```

Reply:
I was trying to work towards centralizing the log group creation.  There is no reason it couldn't be created here.  I could also create the kms_log group in the kms log module.

Reply:
ahh ok, that makes sense to me. I think either way is fine, create the log group with the associated resource/module or centralize the log group creation in one place. 👍